### PR TITLE
Mep12 logo.py

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

File: examples/pylab_examples/logo.py [hunk 1]
```diff
@@ -1,28 +1,30 @@
-#!/usr/bin/env python
 # This file generates the matplotlib web page logo
 
-from __future__ import print_function
```
Comment (WeatherGod) [inline examples/pylab_examples/logo.py]: you can't take this line out if we are going to have a print() in this script.

File: examples/pylab_examples/logo.py [hunk 2]
```diff
@@ -1,28 +1,30 @@
...
+plt.plot(t, x)
+plt.text(0.5, 0.5, 'matplotlib', color='r',
+         fontsize=40, fontname=['Courier', 'Bitstream Vera Sans Mono'],
+         horizontalalignment='center',
+         verticalalignment='center',
+         transform=ax.transAxes,
+         )
+plt.axis([1, 1.72, -60, 10])
+plt.setp(plt.gca(), 'xticklabels', [])
+plt.setp(plt.gca(), 'yticklabels', [])
```
Comment (WeatherGod) [inline examples/pylab_examples/logo.py]: I think we can change these to `plt.gca().set_xticklabels([])` and `plt.gca().set_yticklabels([])`.

**Comment by jenshnielsen** [comment]
Could you change the comment at the top to reflect that this is not the current logo. I.e. to something like "this generates a old version of the matplotlib logo ..."

**Comment by ericmjl** [comment]
requested changes made! ready to merge?

**Comment by jenshnielsen** [comment]
Thanks

**Pull request merged**